addons/payment_test/tests/common.py

```diff
@@ -0,0 +1,78 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.addons.payment import utils as payment_utils
+from odoo.addons.payment_test.controllers.main import PaymentTestController
+from odoo.addons.payment.tests.common import PaymentCommon
+
+
+class PaymentTestCommon(PaymentCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        cls.acquirer = cls._prepare_acquirer(provider='test', update_values={
+            'capture_manually': False, # Only activate manual capture in dedicated tests
+            'fees_active': False, # Only activate fees in dedicated tests
+        })
+
+        cls.PaymentTestController = PaymentTestController()
```

Comment:
Why do you instantiate the Controller ?
You are only using the class to get the class attributes in the tests, never `self.PaymentTestController`